### DISCUSSION TITLE
International students have a verification problem

### DISCUSSION THREAD

**Discussion by richtungspfeil** [Digitaler-Impfnachweis/covpass-android#5] [General]
Students studying in Germany are currently unable to provide proof of their vaccination. They do not have a vaccination certificate or an account registered in Germany in one of the app stores. Unfortunately, in the Android area a change of the account is not possible because it must be held for one year. Can you release the app for the International market? You would enable a return trip of the students to your home country.

**Comment by wkornewald** [comment]
Thanks for your suggestion. I've forwarded this discussion internally.

**Comment by Ein-Tim** [comment]
Anderer Issue: https://github.com/Digitaler-Impfnachweis/documentation/issues/5

**Comment by 10Meisterbaelle** [comment]
There are two ways to verify as vaccinated if you live in another country: You could either use the app from your region (it will probably be accepted) or install the Huawei AppGallery, set your region to Germany and install it from there.

EDIT: This is EU only, you need an EU vaccination certificate

**Comment by martinlicht** [comment]
If nothing else helps:
The Swizz "COVID Certificate" app seems to be available outside of Germany and accepts EU vaccine passports. It automatically displays in English if that's your phone's language.
